### ISSUE TITLE
Redo broken GIFs

### ISSUE THREAD

**Issue by alix-tz** [alix-tz/escriptorium-tutorial#63]
Unfortunately, @FloChiff GIFs don't work on every browsers, we need to find out why, and/or to remake them so that they loop instead of stopping after 1 loop.

**Comment by HugoSchtr** [comment]
On Chrome, gifs seem to be working fine! Do we know which browsers have issues displaying the gifs?

**Comment by alix-tz** [comment]
Yes, I have this problem on Firefox, whether I'm on the local dev server or on the deployed websie

**Comment by HugoSchtr** [comment]
Are all gifs from @FloChiff broken on your firefox? Could you link a walkthrough where you encounter the problem please?

**Comment by FloChiff** [comment]
I just check and on my computer, whether it is with Firefox or Chrome, some GIFs just refuse to loop (I only looked at the manual transcription walkthrough)

**Comment by HugoSchtr** [comment]
Weird, the transcription walkthrough works fine on my computer. 🤔
